Tests/Testo/Widget/ClientFormFieldsTest.php

```
@@ -0,0 +1,93 @@
+ * legacy local-format number (no leading '+') silently fails validation on
+ * every save with no visual cue why. Found 2026-08-31 via a real client
+ * whose stored '07726232648' blocked an unrelated field (postaladdress_id)
+ * from ever saving, since form validation fails as a whole.
+ */
+#[Test]
+final class ClientFormFieldsTest
+{
+    private function fields(): ClientFormFields
+    {
+        /** @var TranslatorInterface&m\MockInterface $translator */
+        $translator = m::mock(TranslatorInterface::class);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Mockery' in method 'fields'. (undefined)

(StaticAccess)